nixos/modules/misc/ids.nix

```diff
@@ -281,8 +281,8 @@
       stanchion = 262;
       riak-cs = 263;
       infinoted = 264;
-      keystone = 265;
-      glance = 266;
+      # keystone = 265; # unused
+      # glance = 266; #unused
```

Comment:
We usually use `unused` when only either group id or user id is unused.

Reply:
Oh, i thought it was for showing that there are still `unused` numbers from old packages in this sections which are up for grabs. i can keep 2 empty lines instead

Reply:
ok, then please that remark.

Reply:
okay done